src/components/NavBar/SignOutButton.tsx

```diff
@@ -1,21 +1,22 @@
+"use client";
```

Comment:
Why do we need "use client" here?

Reply:
I think we need it for `createClientComponentClient`, I'm not totally sure though I'll have a look into it

Reply:
Yep we do :)